org.eclipse.xtext/src/org/eclipse/xtext/nodemodel/impl/InternalNodeModelUtils.java

```diff
@@ -19,7 +19,7 @@
 import com.google.common.collect.Lists;
 
```

Comment:
Update copyright year (I could not add the comment to the right line)

Reply:
Done